tests/test_data_generators.py

```diff
@@ -183,8 +183,8 @@ def test_DataGeneratorAllSpectrums_additional_inputs():
     spectrums = load_processed_spectrums()
     tanimoto_scores_df = get_reference_scores()
 
-    test_cases = [(PrecursorMZFeatureGenerator, ),  # We are using two test cases, one with only one
-                  (PrecursorMZFeatureGenerator, PrecursorMZFeatureGenerator)]  # Normally you would of course have two different ones (but metadata did not allow it)
+    test_cases = [(StandardScaler("precursor_mz", mean=0, std=1000), ),  # We are using two test cases, one with only one
+                  (StandardScaler("precursor_mz", mean=0, std=1000), StandardScaler("precursor_mz", mean=0, std=100))]  # Normally you would of course have two different ones (but metadata did not allow it)
```

Comment:
Above I suggested something like a `NumericalToBinary` generator, which could for instance be used here in the test (m/z >= 500 --> 1, else: 0).

Reply:
The current CategoricalToBinary does not exactly do this. It requires to name every possible input value instead of a range. We can later also easily add NumericalToBinary if we want to. For now I will use the CategoricalToBinary for ionization mode as additional test case.